src/main/webapp/app/controllers/submission/submissionListController.js

```diff
@@ -51,85 +53,142 @@ vireo.controller("SubmissionListController", function (NgTableParams, $controlle
         var start;
 
         var query = function () {
+            var pageNumber = $scope.page.number;
+            var pageCount = $scope.page.count;
+
+            if (angular.isDefined(sessionStorage.getItem("list-page-number")) && sessionStorage.getItem("list-page-number") !== null) {
```

Comment:
Why not check session storage once for each?

Better yet, why check session storage each time the query is issued?

Shouldn't the scope already be updated?

Reply:
A simple oversight.
I can switch to a variable:
```js
   var pageNumber = sessionStorage.getItem("list-page-number");
```

> Shouldn't the scope already be updated?

That is session storage data and may be updated outside of the view.